src/vs/base/browser/ui/selectBox/selectBoxCustom.ts

```diff
@@ -110,6 +110,10 @@ export class SelectBoxList implements ISelectBoxDelegate, IDelegate<ISelectOptio
 		this.selectElement = document.createElement('select');
 		this.selectElement.className = 'monaco-select-box';
 
+		if (selectBoxOptions && selectBoxOptions.ariaLabel) {
+			this.selectElement.setAttribute('aria-label', selectBoxOptions.ariaLabel);
+		}
+
```

Comment:
Should we fix in VScode source code? or should we resolve the issue as external from VSCode

Reply:
We could, I've generally only been resolving as external if its a large amount of changes. This really wasn't that much change.

Reply:
@kburtram do you have any preference?

Reply:
@AbbiePetcht I think if we can fix the bug ourselves then let's go for it.  Though it's might be worth opening an equivalent PR against VS Code so they can pick up the fix upstream.

Reply:
@kburtram do we want to fix it here, and submit a VSCode PR? or just submit a vscode PR and we will merge it in a future merge?